bin/setup

```diff
@@ -12,12 +12,11 @@ echo -e "You may need to enter your root password...\n"
 sudo mkdir -p /etc/resolver
 echo -e "nameserver 127.0.0.1\nport 53" | sudo tee /etc/resolver/dev.gov.uk >/dev/null
 
-echo -e "✅ Configuring dnsmasq [/usr/local/etc/dnsmasq.conf]\n"
-echo "conf-dir=/usr/local/etc/dnsmasq.d,*.conf" > /usr/local/etc/dnsmasq.conf
+echo -e "✅ Configuring dnsmasq [/opt/homebrew/etc/dnsmasq.conf]\n"
```

Comment:
I think it should be possible to make this cross-architecture compatible by using `brew --prefix` instead of hardcoding the path:

```suggestion
echo -e "✅ Configuring dnsmasq [$(brew --prefix)/etc/dnsmasq.conf]\n"
```

Reply:
Ooh, interesting. I'd assumed we'd do a bit of cross-architecture sniffing when we merged into full main, but this is very low-cost, worth doing now. I'll update and confirm it works on this machine, if someone else can then try it on an intel machine.